.github/workflows/build.yml

```diff
@@ -12,6 +12,8 @@ jobs:
 
       - name: Setup .NET Core SDK
         uses: actions/setup-dotnet@v5
+        with:
+          dotnet-version: 10.0.x
```

Comment:
Is this needed? Doesn't setup-dotnet automatically pick up whatever's in global.json?

Reply:
Could have been a temporary issue, let me try and remove